honestcash-v2/src/app/modules/editor/header/header.component.ts

```diff
@@ -0,0 +1,68 @@
+import { Component, OnInit } from '@angular/core';
+import User from '../../../core/models/user';
+import {NgbModal} from '@ng-bootstrap/ng-bootstrap';
+import { EmbeddableEditorComponent } from '../embed/embed.component';
+
+@Component({
+             selector: 'app-header',
+             templateUrl: './header.component.html',
+             styleUrls: ['./header.component.scss']
+           })
+export class HeaderComponent implements OnInit {
+  menuHidden = true;
+  // @todo get it from store instead
+  public user: User;
+
+  constructor(
+    private modalService: NgbModal
+  ) {}
+
+  ngOnInit() {
+    // @todo access from store instead
+    this.user = new User();
```

Comment:
You can subscribe to the user already?

Reply:
no need to subscribe, the store already provides it. the store will be injected in the constructor then user will be assigned via store selector.

Reply:
this is what I meant :)